[2024-04-09T12:31:42.159+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 21120051.1200 scheduled__2024-04-09T12:26:29.101242+00:00 [queued]>
[2024-04-09T12:31:42.169+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 21120051.1200 scheduled__2024-04-09T12:26:29.101242+00:00 [queued]>
[2024-04-09T12:31:42.170+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-04-09T12:31:42.340+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): 1200> on 2024-04-09 12:26:29.101242+00:00
[2024-04-09T12:31:42.346+0000] {standard_task_runner.py:60} INFO - Started process 525 to run task
[2024-04-09T12:31:42.350+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '21120051', '1200', 'scheduled__2024-04-09T12:26:29.101242+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpvmxllhoi']
[2024-04-09T12:31:42.353+0000] {standard_task_runner.py:88} INFO - Job 55: Subtask 1200
[2024-04-09T12:31:42.403+0000] {task_command.py:423} INFO - Running <TaskInstance: 21120051.1200 scheduled__2024-04-09T12:26:29.101242+00:00 [running]> on host 4b0cf5e8e48f
[2024-04-09T12:31:42.491+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='21120051' AIRFLOW_CTX_DAG_ID='21120051' AIRFLOW_CTX_TASK_ID='1200' AIRFLOW_CTX_EXECUTION_DATE='2024-04-09T12:26:29.101242+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-09T12:26:29.101242+00:00'
[2024-04-09T12:31:42.650+0000] {python.py:202} INFO - Done. Returned value was: {'status': 'error', 'message': "'Thông báo Hội thảo định hướng tách ngành cho SV Đại học đại trà Khóa 15 (DHTH15)' not found in data"}
[2024-04-09T12:31:42.677+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=21120051, task_id=1200, execution_date=20240409T122629, start_date=20240409T123142, end_date=20240409T123142
[2024-04-09T12:31:42.722+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-09T12:31:42.751+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
